### DISCUSSION TITLE
h264 or MJPEG preference

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#4949] [Ask A Question] [answered]
I am starting over with my configuration.
If a camera can output either h264 or MJPEG which protocol is preferred?
http://www.spu.co.id/news/11/MJPEG-MPEG4-or-H-264-which-one-is-better

**Comment by NickM-27** [comment, marked as answer]
100% h264 is preferred, otherwise you will just need to have your system do the work of transcoding to h264 anyway
